nesta_ds_utils/plotting.py

```diff
@@ -120,3 +123,99 @@ def save(
 
     if save_svg:
         _save_svg(fig, path, name, scale_factor, driver)
+
+
+def _find_averta() -> str:
+    """Search for averta font, otherwise return 'Helvetica' and raise a warning.
+
+    Returns:
+        str: Return averta installation name, if found. Return 'Helvetica' otherwise.
+    """
+    font = False
+    for font_path in font_manager.findSystemFonts(fontext="ttf"):
+        if "averta" in font_path:
+            font = "averta"
+            break
+        if "Averta" in font_path:
+            font = "Averta"
+            break
+    if not font:
+        warnings.warn("Averta font could not be located. Using 'Helvetica' instead")
+        font = "Helvetica"
+    return font
+
+
+def nesta_theme():
```

Comment:
Should nesta_theme be private? Do we envision people to access it only through setup_theme or is there a reason people would call it directly?

Reply:
That's a good point, I thought about it. I thought maybe at some point we might have multiple themes and users might want to access them to modify them.

Reply:
I think the best way to do that is to have the theme values be in a yaml file so that we're not modifying the code if the theme changes. We could always have multiple yaml files if there are different themes.